main.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = "~> 1.3.3"
+
+  required_providers {
+    aws = {
+      version = "~> 5.0.0"
```

Comment:
Should this be `>=` ?

Reply:
https://developer.hashicorp.com/terraform/tutorials/configuration-language/versions#terraform-version-constraints

![image](https://github.com/madelabs/terraform-aws-ecs-app/assets/1960279/08d0da6b-4194-4976-a2b2-f8fa72268e41)

I think this would keep us in 5.x and not accidentally bump us to 6.x

Reply:
Right, but as of now, greater versions *that we know of* will not break usage of this module, do we only switch to `~>` when we know of breaking changes on newer versions?